.github/workflows/build.yml

```diff
@@ -43,6 +55,8 @@ jobs:
     steps:
       - name: "Checkout directory"
         uses: actions/checkout@v2
+        with:
+          ref: ${{ inputs.build-ref }}
       - name: Generate metadata file
         id: generate-metadata-file
         uses: hashicorp/actions-generate-metadata@v1
```

Comment:
Just FYI - We use this action to generate metadata that is used by the orchestrator in the rest of the pipeline. Currently it uses `GITHUB_SHA` which will be fine based on my comment above - but if we were to allow the release workflow to be triggered on a specific sha - we'd need to update this action to allow the sha to be passed in as a param.

Reply:
Ah OK. The action probably needs to be updated regardless? We need this to be the SHA of the commit with the generated files, not necessarily the latest commit when the release was triggered.